src/components/date-picker/index.js

```diff
@@ -0,0 +1,2 @@
+import DatePicker from './date-picker';
+export default DatePicker;
```

Comment:
Keep it as is though since `webpack.config.babel.js` would blow up if changing this export.